paths.tex

```diff
@@ -255,39 +252,26 @@ \subsection{Windowed edges}
 specific to each edge (as is specified in the definition of an interval network
 temporal graph). We call this graph a \textbf{windowed co-authorship network} or
 simply a \textbf{co-authorship network}. If we specify a universal
-edge-persistence $\Delta t$ such that for all edges $(u,v,t_1,t_2)$ in the
-network, $\Delta t = t_2 - t_1$, then we call this co-authorship network
+edge-persistence $\Delta t$ such that for all edges $e$ in the
+network, $\Delta t = |T(e)|$, then we call this co-authorship network
 \textbf{$\Delta t$-windowed}.
 
-When we consider the above definition of a path it is clearly too
-simplistic, as it does not consider that an edge may cease to exist. So let
-us consider a new definition of a temporal path.
-
-\begin{defn}
-  A \textbf{windowed temporal path} bewteen $v_1$ and $v_n$ is a collection of
-  edges $(v_1,v_2,s_1,t_1),(v_2,v_3,s_2,t_2), \cdots (v_{n-1},v_{n}, s_{n-1},
-  t_{n-1})$ such that $v_1$ and $v_n$ have degree $n$, and the remaining
-  vertices have degree 2.  Most importantly, $t_i \geq s_{i+1}$ for all $i \in
-  [n-2]$.
-\end{defn}
-
-The definitions for fastest and shortest path will be the same as in definition
-\ref{defn:short_fast_path}.
+The definitions for temporal paths will be the same as for infinitely persistent
+edges, \ref{defn:cons_temp_path}. As well as those for fastest and shortest path
+will be the same as in definition \ref{defn:short_fast_path}.
 
-\section{Pairwise Contemporaneity}
+\section{Complete Contemporaneity}
```

Comment:
Perhaps we should call this *Concurrent Contemporaneity* -- for some reason my brain isn't reading *complete* in its intended way.